src/swap/components/SwapSettingsSlippageInput.tsx

```diff
@@ -0,0 +1,123 @@
+import { useCallback, useState } from 'react';
+import { background, border, cn, color, pressable } from '../../styles/theme';
+import type { SwapSettingsSlippageInputReact } from '../types';
+import { useSwapContext } from './SwapProvider';
+
+const slippageSettings = ['Auto', 'Custom'];
```

Comment:
For next PR, `slippageSettings` -> `SLIPPAGE_SETTINGS`, as value that is always constant, it's UPPER_CASE